profiles.yaml

```diff
@@ -0,0 +1,58 @@
+# yaml-language-server: $schema=https://creativeprojects.github.io/resticprofile/jsonschema/config-1.json
+
+version: "1"
+
+groups:
+  ddev:
+    - ddev-db
+    - ddev-files
+
+default:
+  repository: "s3:blr1.digitaloceanspaces.com/axl-backup-contrib-tracker/{{ .Profile.Name }}"
+  insecure-no-password: true
```

Comment:
@hussainweb I think there should be some way of authentication to the bucket at least where Production data i stored.

Reply:
I am also thinking of asking the PMs if there would be any contractual clauses that might/might not allow us to do this. We have one in AMTS I believe

Reply:
> @hussainweb I think there should be some way of authentication to the bucket at least where Production data is stored.

However I think this might not be required if we are doing cleanup before storage

Reply:
THanks. Since we are using AWS tokens from 1Password I think we are good here.